configure.ac

```diff
@@ -2989,6 +2989,10 @@ AC_CHECK_TYPE(mtyp_t,AC_DEFINE(HAVE_MTYP_T,1,
 #include <sys/ipc.h>
 #include <sys/msg.h>])
 
+AC_CHECK_TYPE(cpu_set_t,
+   AC_DEFINE(HAVE_CPU_SET_T,1,[cpu_set_t is defined by the system headers]),
+   ,[#include <sched.h>])
```

Comment:
Should this `#include` be protected with `HAVE_SCHED_H`? We do have that protection in `compat/cpu.h`...

Reply:
It should not have to, but done anyway.